pkg/volume/flexvolume/fake_watcher.go

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
Move this to util too?

Reply:
I decided to leave it in Flexvolume directory because fakes could be different depending on test.

Reply:
ACK